assets/js/modules/analytics/components/dashboard/DashboardAllTrafficWidget/index.js

```diff
@@ -155,71 +155,75 @@ function DashboardAllTrafficWidget() {
 		currentRange,
 	] );
 
-	if ( pieChartError ) {
-		return <ReportError moduleSlug="analytics" error={ pieChartError } />;
-	}
-
-	if ( isZeroReport( pieChartReport ) ) {
-		// TODO: Replace with `props.WidgetReportZero` once legacy dashboard UI has been removed.
-		return <ReportZero moduleSlug="analytics" />;
-	}
+	const isZero = isZeroReport( pieChartReport );
 
 	return (
-		<Widget
-			slug="analyticsAllTraffic"
-			className="googlesitekit-widget--footer-v2"
-			footer={ () => (
-				<SourceLink
-					className="googlesitekit-data-block__source"
-					name={ _x( 'Analytics', 'Service name', 'google-site-kit' ) }
-					href={ serviceReportURL }
-					external
-				/>
-			) }
-			noPadding
-		>
-			<Grid>
-				<Row>
-					<Cell
-						className="googlesitekit-widget--analyticsAllTraffic__totals"
-						lgSize={ 7 }
-						mdSize={ 8 }
-					>
-						<TotalUserCount
-							loaded={ totalUsersLoaded && ! firstLoad }
-							report={ totalUsersReport }
-							error={ totalUsersError }
-							dimensionValue={ dimensionValue }
-						/>
-
-						<UserCountGraph
-							loaded={ userCountGraphLoaded && ! firstLoad }
-							error={ userCountGraphError }
-							report={ userCountGraphReport }
-						/>
-					</Cell>
-
-					<Cell
-						className="googlesitekit-widget--analyticsAllTraffic__dimensions"
-						lgSize={ 5 }
-						mdSize={ 8 }
-					>
-						<DimensionTabs
-							loaded={ ! firstLoad }
-							dimensionName={ dimensionName }
-						/>
-
-						<UserDimensionsPieChart
-							dimensionName={ dimensionName }
-							dimensionValue={ dimensionValue }
-							sourceLink={ serviceReportURL }
-							loaded={ pieChartLoaded && ! firstLoad }
-							report={ pieChartReport }
+		<Fragment>
+			<div style={ { display: pieChartError ? 'block' : 'none' } }>
+				<ReportError moduleSlug="analytics" error={ pieChartError || {} } />
+			</div>
+			<div style={ { display: isZero && ! pieChartError ? 'block' : 'none' } }>
+				{ /* TODO: Replace with `props.WidgetReportZero` once legacy dashboard UI has been removed. */ }
+				<ReportZero moduleSlug="analytics" />
+			</div>
+			<div style={ { display: pieChartError || isZero ? 'none' : 'block' } }>
```

Comment:
Instead of using the `style` attribute here, let's use the `googlesitekit-hidden` class.

Reply:
Ok, looks like these changes in the index.js file aren't needed. So, I have reverted it.